src/runtime/data/kata-collect-data.sh

```diff
@@ -82,14 +86,17 @@ usage()
 	cat <<EOT
 Usage: $script_name [options]
 
-Summary: Collect data about an installation of @PROJECT_NAME@.
+Summary: Collect data about an installation of $project_name.
 
 Description: Run this script as root to obtain a markdown-formatted summary
-  of the environment of the @PROJECT_NAME@ installation. The output of this script
+  of the environment of the $project_name installation. The output of this script
   can be pasted directly into a github issue at the address below:
 
       $issue_url
 
+Commands:
+ show-image-details : outputs the yaml describing the image
```

Comment:
Nit: capitalise acronyms:

```suggestion
 show-image-details : outputs the YAML describing the image
```